events/Settings.txt

```diff
@@ -206,9 +206,9 @@ country_event = {
 	option = {
 		name = ""
 		
-		set_global_flag = unciv_5
-		clr_global_flag = unciv_7
-		clr_global_flag = unciv_10
+		FOR = { set_country_flag = unciv_5 }
```

Comment:
```
any_country = {
    set_country_flag = unciv_5
    clr_country_flag = unciv_7
    clr_country_flag = unciv_10
}
```

The flag needs to be on all tags. Not just the utility tag. If memory serves this also eliminates the need for cleanup in the FOR events -- any_country in events targets all tags in the game, not just those that are existing